js/src/ui/Form/InputAddress/inputAddress.js

```diff
@@ -76,7 +76,7 @@ class InputAddress extends Component {
 
     const props = {};
 
-    if (!readOnly && !disabled) {
+    if (!disabled) {
```

Comment:
Why was this changed? The whole readOnly/disabled is an issue overall for this component, seems that was added due to some issue somewhere and now removed due to some issue here.